dashboard/common/utils.py

```diff
@@ -1,14 +1,13 @@
 import logging
-from decouple import config
 logger = logging.getLogger(__name__)
-
+from django.conf import settings
 
 def get_build_info():
     logger.debug(get_build_info.__name__)
-    git_commit=config("OPENSHIFT_BUILD_COMMIT",default="")
-    build_namespace=config("OPENSHIFT_BUILD_NAMESPACE",default="")
-    git_branch=config("OPENSHIFT_BUILD_REFERENCE",default="master")
-    build_source=config("OPENSHIFT_BUILD_SOURCE",default="")
-    build_name=config("OPENSHIFT_BUILD_NAME",default="")
+    git_commit=settings.ENV.get("OPENSHIFT_BUILD_COMMIT", "")
```

Comment:
I think these are really in the environment so I'm going to have to switch this to an os.getenv.